Sources/System/Process/ProcessID.swift

```diff
@@ -0,0 +1,30 @@
+
+// FIXME(DO NOT MERGE): We need to find a way around this. We want to declare
+// a typealias to a struct from a header, but don't want downstream to import
+// Darwin or the whole header just for that.
```

Comment:
Can we use `Darwin`'s types though? Also, can we do this with our `CSystem` module as well (which is needed for Linux, since `libC` isn't `libSystem`).

Reply:
I think so -- if people only import System, they will still be able to refer to these types by one of their System typealiases (such as the ones in CInterop or RawValue). (They can't spell `Darwin.foo` unless they import it.)

`CSystem` is trickier, because it shouldn't be a public module, so we should continue to import it `@_implementationOnly`.